cmd/generate/generate.go

```diff
@@ -77,6 +77,8 @@ type Schema struct {
 	Binding                        kapi.Binding
 	LimitRangeList                 kapi.LimitRangeList
 	DeleteOptions                  kapi.DeleteOptions
+	CustomObject                   kapi.CustomObject
```

Comment:
These types don't exist in the Kubernetes packages. I'm assuming you've just put them here to follow the other conventions? Not going to work unfortunately as this needs to compile...